jenkins/dockerfiles/gcc-8_arm/Dockerfile

```diff
@@ -20,4 +20,6 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
    libcap-dev:armhf \
    libcap-ng-dev:armhf \
    libelf-dev:armhf \
-   libpopt-dev:armhf
+   libpopt-dev:armhf \
```

Comment:
Please keep the package names in alphabetical order.

Reply:
I will keep, sorry about it! I forgot about your other comment.